src/components/layout/ExternalLink.tsx

```diff
@@ -3,9 +3,9 @@ import { ParentComponent } from "solid-js";
 export const ExternalLink: ParentComponent<{ href: string }> = (props) => {
     return (
         <a target="_blank" rel="noopener noreferrer" href={props.href}>
-            {props.children}{" "}
+            {props.children}
             <svg
-                class="inline-block"
+                class="inline-block pl-0.5"
```

Comment:
I don't think tailwind works like this. what were you trying to fix?

Reply:
spacing between the text and the externalLink children and the upArrow, right now the whitespace string adds an underline over nothing.

![image](https://github.com/MutinyWallet/mutiny-web/assets/108441023/24b896d0-4ac3-4890-b4f2-1d8e6aaf4ceb)

![image](https://github.com/MutinyWallet/mutiny-web/assets/108441023/9eb7350a-1b9b-4604-b0cb-f8a268f99162)

Reply:
No theres a 0.5 https://tailwindcss.com/docs/padding

Reply:
wow TIL I am in fact crazy